agents/unix/meson.build

```diff
@@ -473,6 +482,7 @@ rcf_listener_sources = files(
 )
 
 # Just use the ta dependencies: this seems to be enough
+# Skip external libraries, however
```

Comment:
What is the real cause of these changes? Sorry, I don't believe that it is just some kind optimization. So, I'd like to know real cause. It must be stated in the changeset description.

In general, the changeset breaks definition of the platform libraries to be linked to all built executables for the platform. The goal of these libraries is to add platform dependencies.

Reply:
Sorry, explanations are still unclear. What happens if you just remove the patch?